Libraries/Opc.Ua.Client/CoreClientUtils.cs

```diff
@@ -260,10 +280,22 @@ int discoverTimeout
                             continue;
                         }
 
-                        // skip unsupported security policies
-                        if (SecurityPolicies.GetDisplayName(endpoint.SecurityPolicyUri) == null)
+                        if (configuration != null)
                         {
-                            continue;
+                            // skip unsupported security policies
```

Comment:
I don't knwo where LoadCertificate gets the certs from, but this check should come from the same place? Is it?